scripts/private/Format-Checkboxes.ps1

```diff
@@ -1,4 +1,4 @@
-function Format-Checkboxes() {
+﻿function Format-Checkboxes() {
```

Comment:
Should that not be `Format-Checkboxes` (without the `()`)?

Reply:
I added `()` on purpose on all functions, and not for consistency.

The PowerShell debugger, IntelliSense, and the formatter in vscode was having problems with every file where there was a function that didn't have `()` at the end.
So to be able to format the document on save, and to actually be able to debug the scripts I had to add them. (PowerShell would crash during debugging otherwise).